packages/zoe/src/contractFacet/zcfSeat.js

```diff
@@ -412,8 +412,12 @@ export const createSeatManager = (
           }
         },
         reallocate(/** @type {ZCFSeat[]} */ ...seats) {
-          seats.forEach(assertActive);
-          seats.forEach(assertStagedAllocation);
+          for (const seat of seats) {
+            assertActive(seat);
+          }
+          for (const seat of seats) {
+            assertStagedAllocation(seat);
+          }
```

Comment:
Per conversation above:
```suggestion
          for (const seat of seats) {
            assertActive(seat);
            assertStagedAllocation(seat);
          }
```